automata/regex/parser.py

```diff
@@ -24,10 +25,21 @@
     validate_tokens,
 )
 
+# Add these at the top of the file to define our shorthand character sets
+ASCII_PRINTABLE_CHARS = frozenset(string.printable)
```

Comment:
@eliotwrobson The implication here is that only ASCII characters are deemed as printable characters, but how would that work given that #233 just added support for Unicode characters?

Reply:
I think this only gets used when adding characters from the use of some special character classes? It might be good to have a test case that uses a non-printable character.